app/grandchallenge/workstation_configs/models.py

```diff
@@ -168,6 +182,13 @@ class WorkstationConfig(TitleSlugDescriptionModel, UUIDModel):
         related_name="workstation_default_window_presets",
     )
 
+    image_context = models.CharField(
+        blank=True,
+        max_length=6,
+        default=IMAGE_CONTEXT_DEFAULT,
```

Comment:
It's weird to have a "default" choice item and let this model field be blank. What is the different between "default" and "blank"? If there is no difference, remove `IMAGE_CONTEXT_DEFAULT` as a choice and do not set a default in the model.

Reply:
I had default initially, but it interfered with some testing and it felt weird having a required field. In the end, I picked Harm's initial instinct to add a Default to the enum. However, I do agree a do real blank is nicer.